v0/destinations/freshsales/config.js

```diff
@@ -0,0 +1,22 @@
+const { getMappingConfig } = require("../../util");
+
+const CONFIG_CATEGORIES = {
+  IDENTIFY: {
+    name: "identifyConfig",
+    type: "identify",
+    endpoint: ".myfreshworks.com/crm/sales/api/contacts/upsert",
+    method: "POST"
+  },
+  GROUP: {
+    name: "groupConfig",
+    type: "group",
+    endpoint: ".myfreshworks.com/crm/sales/api/sales_accounts/upsert",
+    method: "POST"
+  }
+};
+
```

Comment:
Aren't we making track calls?

Reply:
Currently some APIs that are required for making a track call are deprecated. It seems freshsales are working on it and are going to update the doc soon. So we need to enhance our destination after that.

Reply:
Segment docs says it supports Identify and track, but its using deprecated APIs(verified from event tester)

Reply:
It seems they've updated their doc with `api/selector/sales_activity_types` api using which we can finish track call implementation. I'll enhance it in the next sprint.

Reply:
Got it.